### PR TITLE
pd-simulator: update import-data case

### PR DESCRIPTION
**Pull request by ZenoTan**
### What problem does this PR solve?

Import data case is a case in pd simulator. Originally, it has 40 regions, and it imports many regions afterwards. Then import regions >> previous regions. The real scenario is, import regions << previous regions. So now it imports regions 10% of previous size.

### What is changed and how it works?

Changed import data case.

### Check List

Tests

- No code

### CODE REVIEW

File: tools/pd-simulator/simulator/cases/import_data.go [hunk 1]
```diff
@@ -40,8 +40,8 @@ func newImportData() *Case {
 		})
 	}
 
-	storeIDs := rand.Perm(3)
-	for i := 0; i < 40; i++ {
+	for i := 0; i < 10000; i++ {
```
Comment (lhy1024) [inline tools/pd-simulator/simulator/cases/import_data.go:43]: can it be configured?

Comment (ZenoTan) [reply tools/pd-simulator/simulator/cases/import_data.go]: Fixed.

**Comment by ZenoTan** [comment]
> Will it be helpful to the previous issue?

Which one...

**Comment by ZenoTan** [comment]
PTAL @nolouch

**Comment by lhy1024** [comment]
> > Will it be helpful to the previous issue?
> 
> Which one...

#1775

File: tools/pd-simulator/simulator/cases/import_data.go [hunk 2]
```diff
@@ -120,21 +112,28 @@ func newImportData() *Case {
...
 		}
 		simutil.Logger.Info("import data information",
 			zap.String("table-leader", tableLeaderLog),
 			zap.String("table-peer", tablePeerLog),
 			zap.String("total-leader", totalLeaderLog),
 			zap.String("total-peer", totalPeerLog))
+		checkCount++
+		dev := 0.0
+		for _, p := range regionProps {
+			dev += (p - 10) * (p - 10) / 100
```
Comment (lhy1024) [inline tools/pd-simulator/simulator/cases/import_data.go:132]: will it still be 10 after region num is configured?

Comment (ZenoTan) [reply tools/pd-simulator/simulator/cases/import_data.go:132]: It means if balanced, store each get 10 % of data, no matter what the total is.

**Comment by ZenoTan** [comment]
> > > Will it be helpful to the previous issue?
> > 
> > 
> > Which one...
> 
> #1775

In fact, after changing to the new scenario, because import data(3 out of 10 peers) makes it less balanced, we don't need to set a stop time, but we should observe the deviation. If schedulers work properly, balance should not be greatly affected by the import.

**Comment by lhy1024** [comment]
> > > > Will it be helpful to the previous issue?
> > > 
> > > 
> > > Which one...
> > 
> > 
> > #1775
> 
> In fact, after changing to the new scenario, because import data(3 out of 10 peers) makes it less balanced, we don't need to set a stop time, but we should observe the deviation. If schedulers work properly, balance should not be greatly affected by the import.

This behavior seems to be different compared to other simulators

File: tools/pd-simulator/simulator/cases/import_data.go (hunk 2 above)
Comment (lhy1024) [reply tools/pd-simulator/simulator/cases/import_data.go:132]: OK

**Comment by ZenoTan** [comment]
> > > > > Will it be helpful to the previous issue?
> > > > 
> > > > 
> > > > Which one...
> > > 
> > > 
> > > #1775
> > 
> > 
> > In fact, after changing to the new scenario, because import data(3 out of 10 peers) makes it less balanced, we don't need to set a stop time, but we should observe the deviation. If schedulers work properly, balance should not be greatly affected by the import.
> 
> This behavior seems to be different compared to other simulators

Yes. It is because the new scenario is an unbalanced import. Indeed, it is trivial to add a stop condition.

**Comment by ZenoTan** [comment]
PTAL @nolouch @lhy1024

**Comment by ZenoTan** [comment]
> LGTM. we also need to manual test.

It will stop at tick 1009 if region number is 10000.

**Pull request merged**